### ISSUE TITLE
Menu bar Mac

### ISSUE THREAD

**Issue by Schwalbi** [mipstian/catch#165]
Please add a function to remove or hide the application icon in the menu bar (Mac)

**Comment by mipstian** [comment]
Hmmm and how do you plan to access Catch's windows then?

**Comment by Schwalbi** [comment]
Ideally, it could be like 1Password for example that has a general app and a quick access app in the menu bar. That way, if I choose to, I can get rid of the menu bar app and look for the app if I need it. Since I don't use it often it is not worth it occupying a menu bar space which is happening with Catch at the moment. 

However, before I wrote this comment I had not realised that the app did not function other than a menu bar app. Personally, I would find it easier to just open an app when needed (for example checking for content in the morning) and then knowing I have closed the app. This is also the case because I might be switching between VPNs and would prefer to only use the app when connected to a VPN. The Stop option works fine but I have found that it has suddenly resumed without any indication from me. I know this is a big ask but if you were planning on going in that direction in the next updates I would appreciate if it was included as an option.  

As a sidenote, the app works great. Thank you!
